.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '00 19 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-issue-message: |
+            This issue is marked as stale due to requiring feedback for 30 days. Please respond within 5 days or this issue will be closed due to inactivity.
+          close-issue-message: |
+            This issue was closed due to inactivity. If you can still reproduce this bug, please comment with the following information:
+            1. Full output of running `msbuild --version` and `dotnet --version` (if applicable) on a developer command prompt.
+            2. Steps to reproduce your scenario.
+            3. Any other notes that might help in the investigation.
+          days-before-stale: 30
+          days-before-close: 14
+          stale-issue-label: stale
```

Comment:
Should we turn on `remove-stale-when-updated`? I tend to think so: even if the OP has gone away someone else could help us chase an issue.

Reply:
Sometimes the original author has moved on to different things (e.g. switched projects or company) so tying updates to the original submitter might be tricky

Reply:
Remove-stale-when-updated is defaulted to true, I don't think there's a way to specify "remove-stale-when-author-updated". I think it's fine to un-stale when anyone responds.